DynamicProgramming/EditDistance.java

```diff
@@ -22,11 +23,13 @@
 
 public class EditDistance {
 
-    public static int minDistance(String word1, String word2) {
-        int len1 = word1.length();
-        int len2 = word2.length();
+    @SuppressWarnings({"cast.unsafe", "array.access.unsafe.high"})
+    public static int minDistance(@MinLen(0) String word1, @MinLen(0) String word2) {
```

Comment:
`@MinLen(0)` is meaningless - all strings are at least length 0.

Reply:
I removed this annotation.